balena/settings.py

```diff
@@ -31,14 +31,19 @@ class Settings(object):
     CONFIG_FILENAME = 'balena.cfg'
     DEFAULT_SETTING_KEYS = set(['pine_endpoint', 'api_endpoint', 'api_version',
                                 'data_directory', 'image_cache_time',
-                                'token_refresh_interval', 'cache_directory', 'timeout'])
+                                'token_refresh_interval', 'cache_directory', 'timeout',
+                                'image_maker_endpoint', 'image_maker_endpoint_version',
+                                'device_actions_endpoint_version'])
 
     _setting = {
         # These are default config values to write default config file.
         # All values here must be in string format otherwise there will be error when write config file.
         'pine_endpoint': 'https://api.balena-cloud.com/v5/',
         'api_endpoint': 'https://api.balena-cloud.com/',
         'api_version': 'v5',
+        'image_maker_endpoint': 'https://img.balena-cloud.com/api/v1/',
```

Comment:
if all these are `.balena-cloud.com` for the endpoints, would it make sense to break it out to a single variable (the domain name), and use that for the endpoints with just the subdomains set there?

Reply:
@imrehg I think these endpoints are essentially separate (for old SDK versions, they don't have same domain or in case of Open Balena, they can be anything) so I don't want to add unnecessary complexity here :)